backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -32,22 +32,9 @@ nav.menu {
 }
 
 .admin-nav-header {
-  padding: 15px;
-  height: 58px; // height of .page-title
-  box-sizing: content-box;
-  background-color: $color-1;
+  padding: 14px 0 8px;
```

Comment:
We should also use the (to be introduced) `$main-header-height` variable to set the height on the `.main-header`

```scss
$main-header-height:             75px !default;
```

```scss
.main-header {
  ...
  height: $main-header-height;
  ...
}
```

That fixes the height for empty main headers, in the login screen for instance

### Before

![user sessions 2017-04-05 15-24-36](https://cloud.githubusercontent.com/assets/42868/24707471/ff9b0b0c-1a13-11e7-8b79-c2b945ec93c0.jpg)

### After

![user sessions 2017-04-05 15-25-09](https://cloud.githubusercontent.com/assets/42868/24707492/1389ab78-1a14-11e7-915a-b845870beb59.jpg)